boilerplate/tsconfig.json

```diff
@@ -31,6 +31,6 @@
       "module": "commonjs"
     }
   },
-  "include": ["index.js", "App.tsx", "app", "types", "plugins", "app.config.ts"],
+  "include": ["App.tsx", "app", "types", "plugins", "app.config.ts"],
```

Comment:
I think we've updated this since and this change doesn't apply anymore. Updating the file tree int he README does seem like a good idea, though, I think that's pretty out of date. @andrewsixsixsix would you like to update this PR? If not we can follow up on that.

Reply:
@lindboe Feel free to close it if the changes I made are not relevant